### PR TITLE
Feature/robot  code improvements

### PR DESCRIPTION
**Pull request by skristem**
# Critical Changes
Updated Test scripts as config changes are made to scratch orgs to match trial org
# Changes

# Issues Closed

# New Metadata

# Deleted Metadata

### CODE REVIEW

File: tests/browser/gaus/assign_gau_to_opportunity.robot [hunk 1]
```diff
@@ -13,19 +13,18 @@ Assign GAU to Opportunity
...
-    Select Search    General Accounting Unit 1    &{gau2}[Name] 
-    #sleep    2   
+    Select Search    General Accounting Unit 1    &{gau2}[Name]  
     Add GAU Allocation    Amount 1    20
-    Click Save    GAU
+    Click Button With Value    Save
+    Sleep    2
+    Select Window
+    Page Scroll To Locator    record.related.check_occurrence    GAU Allocations
     Verify Occurrence    GAU Allocations    2
```
Comment (davisagli) [inline tests/browser/gaus/assign_gau_to_opportunity.robot]: Looks like this one failed: https://mrbelvedereci.herokuapp.com/tests/result/31661067

File: tests/browser/donations_payments/create_donation_and_payment.robot [hunk 2]
```diff
@@ -17,12 +17,11 @@ Create Donation from a Contact
...
+    Select Tab    Related
     ${opp_name}    Get Main Header 
     Set Global Variable      ${opp_name}
     Load Related List    Payments
-    Select Related Dropdown    Payments
-    Wait For Locator    link-title    Schedule Payments
-    Click Link    link=Schedule Payments
+    Click Special Related List Button  Payments    Schedule Payments
     Wait For Locator    frame    Create one or more Payments for this Opportunity
     Select Frame with Title    Create one or more Payments for this Opportunity
```
Comment (davisagli) [inline tests/browser/donations_payments/create_donation_and_payment.robot]: Looks like this one failed here: https://mrbelvedereci.herokuapp.com/tests/result/31661060

File: tests/NPSP.py [hunk 3]
```diff
@@ -16,6 +16,7 @@
 from cumulusci.robotframework.utils import selenium_retry
 import sys
 from email.mime import text
+from _socket import timeout
```
Comment (davisagli) [inline tests/NPSP.py]: This import is unused and should be removed.

Comment (skristem) [reply tests/NPSP.py]: I think this got added when I used timeout= 60 on my keyword "wait for locator"

Comment (davisagli) [reply tests/NPSP.py]: _socket.timeout is a class from the Python stdlib but you're not using that class, you're just passing timeout as the name of an argument which is set to 60. So you don't need the import.

**Pull request merged**